examples/ComponentHosting/Podfile

```diff
@@ -0,0 +1,6 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '7.0'
+target 'Sample' do
+  pod 'ComponentKit', :git => 'https://github.com/facebook/componentkit.git', :commit => '115d2012580f17276b5383cd1dad2b306b24aeb1'
```

Comment:
Per our discussion: We probably should reference the latest public release (latest tag) and check-in `Podfile.lock` here as well, instead of referencing a specific commit.